core/src/main/java/com/epam/demo/Utils.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.demo;
+
+import java.util.Arrays;
+
+
+public class Utils {
+
```

Comment:
all code must follow common rules like number of lines between methods, number of lines between last import and class name, the line to put [opening curly brace](https://en.wikipedia.org/wiki/Indentation_style) (Ex. csharp (Allman) or egyptian (K&R) style, ), count of spaces that replaced with tab etc. That way please pay attention: you have a blank line between class definition and first method here but no such line in `App` class. Please select a style you want to use in your code. 
In enterprise solutions you'll face with hundreds/thousands of repositories and may happen that repositories you use in scope of one project can have unsimilar code style.